examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>I am not sure, if that is ParallelBlock. Maybe we just say that we build the Model by chaining concats with mlp_block and prediction_Task</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>